src/lib/consensus/proof_of_stake.ml

```diff
@@ -1045,80 +1048,100 @@ module Make (Inputs : Inputs_intf) :
     let open Consensus_state in
     let open Epoch_data in
     let logger = Logger.child logger "proof_of_stake" in
-    let string_of_option = function
-    | `Take -> "Take"
-    | `Keep -> "Keep"
+    let string_of_option = function `Take -> "Take" | `Keep -> "Keep" in
```

Comment:
I know you didn't write this in this PR but can you change this function to string_of_choice